InnerEye/ML/lightning_models.py

```diff
@@ -108,6 +110,7 @@ def compute_metrics(self, cropped_sample: CroppedSample, segmentation: torch.Ten
         Computes and stores all metrics coming out of a single training step.
         :param cropped_sample: The batched image crops used for training or validation.
         :param segmentation: The segmentation that was produced by the model.
+        :param is_training: Is a training crop
```

Comment:
Please add descriptions for both true and false. Otherwise, people need to magically know what the other branch means.